### ISSUE TITLE
Phase 4: CSS Design

### ISSUE THREAD

**Issue by Nicholas-Lawson** [sukhbirsekhon/Hobbyistic#82]
CSS DESIGN
Phase 4 involves the design of the External Links Widget Page. All pages follow the basic web design principles of
Contrast, Repetition, Alignment, Proximity.

Contrast implementations: Basic color contrast from black and white, font-weight choices, and font-sizes allows for a significant contrast to the webpage.

Repetition implementations: Each task follows similar choice of color, font, and alignment.

Alignment implementations: Each task is aligned right with an edit icon and checkbox.

Proximity implementations: Each task item is evenly spaced out to provide a nice organized look for anyone that wants to manage several list tasks within the webpage.

These implementations were considered from the very beginning as there was a wireframe created.

![Image](https://user-images.githubusercontent.com/68287060/228683287-29b3fe15-aa59-4ec1-9934-9cac88377b76.png)
